components/greaselion/browser/greaselion_service_impl.h

```diff
@@ -46,6 +46,7 @@ class GreaselionServiceImpl : public GreaselionService {
   void SetFeatureEnabled(GreaselionFeature feature, bool enabled) override;
   void UpdateInstalledExtensions() override;
   bool IsGreaselionExtension(const std::string& id) override;
+  std::vector<extensions::ExtensionId> GetExtensionIds() override;
```

Comment:
Please make this private and make the browser test a friend class.

Reply:
Fixed.